components/usage/pkg/server/server.go

```diff
@@ -133,3 +136,7 @@ func registerGRPCServices(srv *baseserver.Server, conn *gorm.DB, stripeClient *s
 	}
 	return nil
 }
+
+func registerHttpHandlers(srv *baseserver.Server, h *stripe.WebhookHandler) {
+	srv.HTTPMux().HandleFunc("/webhook", h.Handle)
```

Comment:
I'd recommend prefixing this with something like `/stripe/invoices/webhook`. This makes it easier to debug when you see just a request HTTP path without knowing what it does.

Reply:
b0d3d891b